apps/openmw/mwrender/actoranimation.cpp

```diff
@@ -51,6 +63,302 @@ ActorAnimation::~ActorAnimation()
     {
         mInsert->removeChild(iter->second);
     }
+
+    mScabbard.reset();
```

Comment:
Is there a purpose to call reset here? C++ compilers generate code to autmatically destruct all members.